fuzz/Cargo.toml

```diff
@@ -0,0 +1,97 @@
+[package]
+name = "gitoxide-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+git-actor = {path= "../git-actor"}
```

Comment:
Is there a need to list every single crate here? I'd expect this dependency list to only include crates which are fuzzed. 

Limiting the list also has benefits for compile times.
Thanks for the clarification.

Reply:
Yeah, that was the plan. It was easier for me to just `ls -1 git-*` + some regex to get all the dependencies. But I agree that this should just be the dependencies that are actually needed.